resteasy-feature-pack/galleon-feature-pack/pom.xml

```diff
@@ -96,6 +96,12 @@
                                     </includes>
                                     <filtering>true</filtering>
                                 </resource>
+                                <resource>
+                                    <directory>${common.resources.directory}</directory>
+                                    <excludes>
```

Comment:
Why do you need to exclude it?

Reply:
Just a copy/paste/typo error. I'll change it to `<includes>`.